### PR TITLE
Add the iMX6ullevk board flashing instructions

### PR DESCRIPTION
**Pull request by Tim-Anderson**
Signed-off-by: Tim Anderson <tim.anderson@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 283b718e03321b11ae4753a0be4cf1fb0e748308 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/743/docs/artifacts/html/index.html

File: source/reference-manual/boards/imx6-prepare.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+-----------
+
+Ensure you replace the ``<factory>`` placeholder below with the name of your
+Factory.
+
+Download necessary files from ``https://app.foundries.io/factories/<factory/targets``
+
+#. Click the latest Target with the :guilabel:`platform-devel` trigger.
+
+   .. figure:: /_static/boards/generic-steps-1.png
```
Comment (mike-scott) [inline source/reference-manual/boards/imx6-prepare.rst:11]: I just noticed this generic image actually shows a RaspberryPi target.  I'm not sure if that's confusing.  Maybe we can crop it tighter to remove that?

Comment (Tim-Anderson) [reply source/reference-manual/boards/imx6-prepare.rst:11]: this is used on all the boards so I would suggest we put it in as a separate PR

Comment (mike-scott) [reply source/reference-manual/boards/imx6-prepare.rst:11]: +1

**Review by mike-scott — APPROVED** [review]
LGTM

**Review by dagriego — COMMENTED** [review]
LGTM

**Pull request merged**